src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let res = 0;
+
+  for (let i = 1; i <= days; i++) {
+    res += 40;
+  }
+
+  return days >= 7 ? res - 50 : days >= 3 ? res - 20 : res;
```

Comment:
you should write clear and understandable code 🙂
will be better to use ternary operator only in case when you need to store one of two values depending on some condition

in this task will be more understandable to use if / else statement

and for now 50, 40, 20 - magic numbers
create variables for them

each of this rules also are on checklist, don't forget to check it)